debian/changelog

```diff
@@ -1,3 +1,9 @@
+repowerd (2020.10+ubports) xenial; urgency=medium
```

Comment:
Please only modify the version number after the plus sign, if the package is not our origin. like ubports1, ubports2 etc. Otherwise we run into problems when we update to a higher upstream version.

Reply:
well "we" are upstream, but we should do proper releases at some point. i'll remove this for now and let jenkins handle the version